### ISSUE TITLE
Kalenderdarstellung 

### ISSUE THREAD

**Issue by xwolfde** [RRZE-Webteam/fau-events#2]
wie beim Blogsystem auch Monats und Wochenkalender

**Comment by cibsell** [comment]
Wie ist denn hier der Stand der Dinge?

**Comment by rvdforst** [comment]
Eine Darstellung des Kalenders in Wochen- und Monatsansicht ist noch im Testbereich. Ich hoffe, dass die kommende Woche das Shortcode freigegeben wird.

**Comment by cibsell** [comment]
Ich habe gesehen, dass es jetzt eine Kalenderansicht gibt. Danke dafür!

Dazu drei Fragen:

1. Bei mir werden die Termine nun mit einem neongelben Punkt bzw. Kalendericon angezeigt, siehe https://www.nat.fau.de/schule/veranstaltungen/?calendar=liste . Kann man die Farbe in das Fakultätsgrün ändern?

2. Kann man standardmäßig die Monatsansicht einblenden? 

3. Kann der Titel in der Kalenderansicht bei den Punkten / Linien bitte eingeblendet werden? Bei Tages- und Wochenansicht werden nur lange Balken eingeblendet, die auch beim Hovern keine Informationen bieten. Man muss erst darauf klicken, damit man den lesen kann, was sich hinter dem markierten Feld verbirgt, siehe: https://www.nat.fau.de/schule/veranstaltungen/?calendar=tag_2016-03-29 Bei der Monatsansicht wäre es auch sinnvoll, wenn der Titel des Events erscheint. Die Benutzer möchten sicher nicht über jeden einzelnen Termin hovern, um zu erfahren, um welchen Termin es sich handelt.

**Comment by rvdforst** [comment]
1. Die Farbe der Terminkategorien kann man unter Termine / Kategorien ändern.
2. Nein, aber das kommt noch (@todo).
3. Bei Tages- und Wochenansicht muss der Titel auch angezeigt werden. Wahrscheinlich es geht hier um einen Fehler (@fix). Bei der Montansicht ist aufwendig den Termintitel anzupassen wenn z.B. 5 oder mehrere Termine den gleichen Tag nutzen. Ich werde mal schauen ob für den Fall eine vernünftige Lösung finde (@todo)

**Comment by cibsell** [comment]
Danke!

Wir haben tatsächlich auch Tage, an denen mehrere Termine gleichzeitig stattfinden, z. B. im Studierenden-Kalender, in dem die Erstsemesterbegrüßungen eingetragen werden (ca. 10 zur gleichen Zeit). Oftmals finden die Sitzungen der Departmentsversammlungen / Kollegialen Leitungen am gleichen Tag statt, so dass eine Häufung von >5 Terminen öfters vorkommen wird.

**Comment by rvdforst** [comment]
Übrigens: Termine wie „Chemistry on tour 2016 - Wir kommen direkt in Ihre Schule“ von 29.02.2016 00:00 bis 12.03.2016 00:00 müssen als ganztägige Termine eingetragen werden sonst werden diese Termine beim Import nicht richtig zugeordnet.

**Comment by cibsell** [comment]
Wie meinen Sie das? Soll der Termin für jeden Tag einzeln eingetragen werden?!

**Comment by rvdforst** [comment]
Nein, der Termin soll für jeden Tag nicht einzeln eingetragen werden. Bei der Erstellung des Termins müssen Sie einfach die Option "Ganztägig" einchecken und das Start- und Enddatum des Termins eingeben.

**Comment by cibsell** [comment]
So hatte ich das auch gemacht...

![terminscreenshot](https://cloud.githubusercontent.com/assets/8234890/13947880/478a4c4c-f01d-11e5-8eb5-4c2c07e94b8f.jpg)

Das ist doch so, wie es sein soll, oder?

**Comment by rvdforst** [comment]
Anscheinend wurde der Termin richtig eingetragen aber aus irgendwelchem Grund der Kalender-Feed enthält keine "all-day-event"-Eigenschaften. Ob der Outlook (welche Version?) bzw. der Exchange-Server die Ursache ist das weiß ich nicht genau (todo).

**Comment by cibsell** [comment]
Mir ist gerade aufgefallen, dass der Abonnement-Link nicht mehr angezeigt wird.

**Comment by rvdforst** [comment]
Sorry aber die Funktion war nicht definiert. Ab sofort wenn der Parameter "abonnement_link" explizit auf 1 festgelegt ist, wird einen Abonnement-Button (neben die Ansichts-Buttons) angezeigt.
